modules/concurrency/src/io_ring/timer.cpp

```diff
@@ -39,8 +38,7 @@ void Timer::Operation::prepare(::io_uring_sqe* sqe) const {
 
 void Timer::Operation::handleCompletion(::io_uring_cqe* cqe) const {
   if (cqe->res < 0 && cqe->res != -ETIME) {
-    heph::panic(
-        fmt::format("timer failed: {}", std::error_code(-cqe->res, std::system_category()).message()));
+    heph::panic("timer failed: {}", std::error_code(-cqe->res, std::system_category()).message());
```

Comment:
while you are at it, would you mind removing all the `heph::` they are not needed given that we are inside that namespace